objects/rct2ww/ride/rct2.ww.italypor.json

```diff
@@ -413,7 +413,7 @@
             "pt-BR": "Polícia Italiana"
         },
         "description": {
-            "en-GB": "Riders ride in pairs of seats rotating around the ends of three long rotating arms",
+            "en-GB": "Riders ride in pairs in Italian police car replicas and rotate in circles",
```

Comment:
Since this is the description that appears in the New Ride menu, I think it should mention the three long rotating arms.